src/pages/specimen.js

```diff
@@ -49,20 +49,26 @@ export const specimenTemplate = async (data, formData) => {
         const siteAcronym = getSiteAcronym();
                 
         if(siteLocations[workflow] && siteLocations[workflow][siteAcronym]) {
+
+            
+            // For the purposes of 1008 we are filtering out some locations.
+            // This will require more discussion for a long-term implementation
+            let siteLocationArray = siteLocations[workflow][siteAcronym]; // Form of [{location, concept}]
+            siteLocationArray = siteLocationArray.filter(loc => ['River East', 'South Loop', 'Orland Park', 'Henry Ford West Bloomfield Hospital', 'Henry Ford Medical Center-Fairlane'].indexOf(loc.location) === -1);
```

Comment:
```suggestion
            siteLocationArray = siteLocationArray.filter(loc => !['River East', 'South Loop', 'Orland Park', 'Henry Ford West Bloomfield Hospital', 'Henry Ford Medical Center-Fairlane'].includes(loc.location));
```

Both your original code and the suggested code works. We can also assign the array of deprecated locations to a variable. The variable can be named `deprecatedLocations` and can be exported from the `shared.js` file. (Optional changes)